src/SuperDumpService/Startup.cs

```diff
@@ -179,44 +182,26 @@ public class Startup {
 				Task.Run(async () => await app.ApplicationServices.GetService<JiraIssueRepository>().Populate());
 			}
 
-			// configure Logger
-			loggerFactory.AddConsole(config.GetSection("Logging"));
-
-			var fileLogConfig = config.GetSection("FileLogging");
-			var logPath = Path.GetDirectoryName(fileLogConfig.GetValue<string>("PathFormat"));
-			Directory.CreateDirectory(logPath);
-			loggerFactory.AddFile(config.GetSection("FileLogging"));
-
-			if (settings.Value.UseAllRequestLogging) {
-				loggerFactory.AddFile(config.GetSection("RequestFileLogging"));
-			}
-
-			loggerFactory.AddDebug();
-
-			oneAgentSdk.SetLoggingCallback(new DynatraceSdkLogger(loggerFactory.CreateLogger<DynatraceSdkLogger>()));
-
 			if (settings.Value.UseHttpsRedirection) {
 				app.UseHttpsRedirection();
 			}
+
+			app.UseStaticFiles();
+			app.UseRouting();
+
 			if (settings.Value.UseLdapAuthentication) {
 				app.UseAuthentication();
+				app.UseAuthorization();
 				app.UseSwaggerAuthorizationMiddleware(authorizationHelper);
 			} else {
+				app.UseAuthorization();
 				app.MapWhen(context => context.Request.Path.StartsWithSegments("/Login") || context.Request.Path.StartsWithSegments("/api/Token"),
 					appBuilder => appBuilder.Run(async context => {
 						context.Response.StatusCode = 404;
 						await context.Response.WriteAsync("");
 					}));
 			}
 
-			if (settings.Value.UseAllRequestLogging) {
-				ILogger logger = loggerFactory.CreateLogger("SuperDumpServiceRequests");
-				app.Use(async (context, next) => {
-					logger.LogRequest(context);
-					await next.Invoke();
-				});
-			}
```

Comment:
This is still needed for the request logging.

Reply:
Good catch. Thank you.

Fixed it.